liquibase/changeLogs/capture/tables/datumConvertedValues/datumConvertedValues.sql

```diff
@@ -0,0 +1,12 @@
+create table ${AQTS_SCHEMA_NAME}.datum_converted_values
+(datum_converted_values_id                int8 NOT NULL GENERATED BY DEFAULT AS IDENTITY,
+json_data_id                              int8 NOT NULL,
+field_visit_identifier                    text NOT NULL,
+target_datum                              text NOT NULL,
+unit                                      text NOT NULL,
+numeric_value                             numeric NOT NULL,
```

Comment:
I don't think we want to carry forward the `numeric_value` in these etls, but rather just stick with the text type `display_value` below.  That way we have a lesser chance of accidentally mutating the AQTS number somewhere along the line, and we just rely on the `display_value` as the source of truth.